examples/plot_newfitting_comparison_oldnew.py

```diff
@@ -0,0 +1,221 @@
+
+begin_time_mark = time.time()
+
+s_best, best = sf.fit_spectrum(
+    s_exp.take("radiance"),
+    model=LTEModel_withslitnorm,
+    fit_parameters={
+        "Tgas": 300,
+        # "offset": 0
+    },
+    bounds={
+        "Tgas": [300, 2000],
+        # "offset": [-1, 1],
+    },
+    plot=True,
+    solver_options={
+        "maxiter": 15,  # 👈 increase to let the fit converge
+        "ftol": 1e-15,
+    },
+    verbose=2,
+)
+
+end_time_mark = time.time()
+
+plot_diff(s_exp, s_best)
+
+# Information to report later in comparison result
+oldfitting_residual = best.fun
+oldfitting_loops = best.nfev
+oldfitting_time = end_time_mark - begin_time_mark
+
+# ------------------------------------------------------------------------------------------------------------ #
+
+
+# -------------------------------------------- NEW FITTING MODULE -------------------------------------------- #
+
+
+# ------------------------------------ Step 1. Load experimental spectrum ------------------------------------ #
+
+
+# Load an experimental spectrum. You can prepare yours, or fetch one of them in the radis/test/files directory.
+my_spec = getTestFile("CO2_measured_spectrum_4-5um.spec")
+s_experimental = load_spec(my_spec).offset(-0.2, "nm")
+
+
+# ------------------------------------ Step 2. Fill ground-truths and data ------------------------------------ #
+
+
+# Experimental conditions which will be used for spectrum modeling. Basically, these are known ground-truths.
+experimental_conditions = {
+    "molecule": "CO2",  # Molecule ID
+    "isotope": "1,2",  # Isotope ID, can have multiple at once
+    "wmin": 4167,  # Starting wavelength/wavenumber to be cropped out from the original experimental spectrum.
+    "wmax": 4180,  # Ending wavelength/wavenumber for the cropping range.
+    "wunit": "nm",  # Accompanying unit of those 2 wavelengths/wavenumbers above.
+    "mole_fraction": 1,  # Species mole fraction, from 0 to 1.
+    "pressure": 1 * 1e-3,  # Partial pressure of gas, in "bar" unit.
+    "path_length": 10,  # Experimental path length, in "cm" unit.
+    "slit": "1.4 nm",  # Experimental slit, must be a blank space separating slit amount and unit.
+    "wstep": 0.001,  # Resolution of wavenumber grid, in cm-1.
+    "databank": "hitran",  # Databank used for the spectrum calculation. Must be stated.
+}
+
+# List of parameters to be fitted, accompanied by their initial values.
+fit_parameters = {
+    "Tgas": 1150,  # Gas temperature, in K.
+}
+
+# List of bounding ranges applied for those fit parameters above.
+# You can skip this step and let it use default bounding ranges, but this is not recommended.
+# Bounding range must be at format [<lower bound>, <upper bound>].
+bounding_ranges = {
+    "Tgas": [
+        300,
+        2000,
+    ],
+}
+
+# Fitting pipeline setups.
+fit_properties = {
+    "method": "lbfgsb",  # Preferred fitting method from the 17 confirmed methods of LMFIT stated in week 4 blog. By default, "leastsq".
```

Comment:
good to mention this ! 

How can we find the other methods ? Can you share this in the docstrings maybe ? 

Btw you can have docstrings appearing nicely as text (with links) in the Gallery example using this format : 

```
# %% 
# Fitting pipeline setups : 
# here this text will render as rst ; including links `like this one <https://...>`__
```

Reply:
I will share the list of 17 available methods in the original example of `plot_newfitting_Tgas.py` (the simplest one, serving as first example).

Also, thank you for the RST with links. I will apply it throughout my docstrings.

Reply:
Add link to lmfit function